sklearn/covariance/tests/test_covariance.py

```diff
@@ -196,6 +192,18 @@ def test_ledoit_wolf():
     assert(lw.precision_ is None)
 
 
+def test_ledoit_wolf_large():
```

Comment:
I could also test this by changing the block_size to something small and then a less large dataset is needed. That wouldn't test the default value, but maybe it is enough.